cmake/CMakeAM.cmake

```diff
@@ -249,12 +249,24 @@ function(parse_automake AM_FILE_NAME)
     list(APPEND AM_PROGRAMS ${AM_TARGET})
   endforeach()
 
+  set(AM_MANS)
+  foreach(AM_MAN ${dist_man1_MANS})
```

Comment:
My bad, that should have been man6, not man1. But you should probably evaluate both, because of xoreos-tools.

But also, the man pages aren't installed in the correct directory. CMake installs them into `man/xoreos.6` now, but it should be `man/man6/xoreos.6`. Compare make install from automake with make install from cmake.